### PR TITLE
Moving module instance number control to `Workbench` and clearing layout on unmount in `App`

### PR DESCRIPTION
**Pull request by rubenthoms**
This fixes issues with React 18 calling `useEffect` in `App` twice and adding data channels etc. twice.

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -14,14 +14,20 @@ const layout: LayoutElement[] = [];
 
 function App() {
     const workbench = new Workbench();
+    console.debug("render app");
```
Comment (sigurdp) [inline frontend/src/App.tsx]: Remove logging

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -14,14 +14,20 @@ const layout: LayoutElement[] = [];
...
 
     React.useEffect(() => {
+        console.debug("app useEffect");
```
Comment (sigurdp) [inline frontend/src/App.tsx]: Remove logging

**Review by sigurdp — APPROVED** [review]
Just remove the console.log() calls 👍

**Comment by rubenthoms** [comment]
> Just remove the console.log() calls 👍

Hehe, already outdated ;)

**Pull request merged**